examples/standalone/02_hello_window/src/main.rs

```diff
@@ -0,0 +1,174 @@
+use std::sync::Arc;
+
+use winit::application::ApplicationHandler;
+use winit::event::WindowEvent;
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
+use winit::window::{Window, WindowId};
+
+struct State {
+    window: Arc<Window>,
+    device: wgpu::Device,
+    queue: wgpu::Queue,
+    size: winit::dpi::PhysicalSize<u32>,
+    surface: wgpu::Surface<'static>,
+    format: wgpu::TextureFormat,
+}
+
+impl State {
+    async fn new(window: Arc<Window>) -> State {
+        let instance = wgpu::Instance::new(&wgpu::InstanceDescriptor::default());
+        let adapter = instance
+            .request_adapter(&wgpu::RequestAdapterOptions::default())
+            .await
+            .unwrap();
+        let (device, queue) = adapter
+            .request_device(
+                &wgpu::DeviceDescriptor::default(),
+                None, // Trace path
+            )
+            .await
+            .unwrap();
+
+        let size = window.inner_size();
+
+        let surface = instance.create_surface(window.clone()).unwrap();
+        let cap = surface.get_capabilities(&adapter);
+        // If we don't add srgb suffix here in most cases the image
+        // rendered will not be "gamma correct".
+        let format = cap.formats[0].add_srgb_suffix();
```

Comment:
This isn't the correct pattern.

You want to:
- Get the capabilities from the surface.
- Use the preferred surface format `cap.formats[0]` directly to configure the surface.
- Get a the view format by calling `surface_format.add_srgb_suffix()`. Add this to the view_formats list on the surface.
- Create a texture view with the view format.

This will work on all backends, no matter if they support srgb surfaces or not